src/common/options.cc

```diff
@@ -610,101 +634,120 @@ std::vector<Option> get_global_options() {
     // unmodified
     Option("clog_to_monitors", Option::TYPE_STR, Option::LEVEL_ADVANCED)
     .set_default("default=true")
+    .set_flag(Option::FLAG_RUNTIME)
```

Comment:
http://pulpito.ceph.com/kchai-2019-04-19_15:53:30-rados-wip-kefu-testing-2019-04-18-1100-distro-basic-smithi/3866955/

Reply:
updated the test